source/unified-test-format/unified-test-format.md

```diff
@@ -3090,6 +3100,32 @@ impossible to guarantee that an existing transaction will not block test executi
 SHOULD either ignore Unauthorized(13) command failures or avoid calling `killAllSessions` altogether when connected to
 Atlas (e.g. by detecting `mongodb.net` in the hostname or allowing the test runner to be configured externally).
 
+##### MigrationConflict Errors on Sharded Clusters
+
+Following [SERVER-82353](https://jira.mongodb.org/browse/SERVER-82353), drivers may encounter MigrationConflict errors
+when executing transactions on sharded clusters due to an outdated cluster time. For example:
+
+```
+Transaction fe0b9a7d-4b64-4e38-aad2-4c7dd5ab0c2b - 47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU= -  - :1 was aborted on statement 0 due to: a non-retryable snapshot error :: caused by :: Encountered error from localhost:27217 during a transaction :: caused by :: Database transaction-tests has undergone a catalog change operation at time Timestamp(1708062645, 94) and no longer satisfies the requirements for the current transaction which requires Timestamp(1708062645, 93). Transaction will be aborted.
+```
+
+To work around this issue, a test runner MUST collect the latest cluster time from processing
+[initialData](#initialData), which is executed using the internal MongoClient, and use it to advance the cluster of time
+of every session entity created during the test, by either [createEntities](#createentities) or the
+[special test operation](#operation_createEntities)). This will ensure that the cluster time is gossipped to any other
+mongo hosts that may be used to execute a transaction during the test.
+
+Depending on the test runner implementation, the cluster time may be collected in various ways. For example:
+
+- Executing a `ping` command against the same mongos host used for [initialData](#initialData) and read the
+  `$clusterTime` response field
+- Execute all operations for [initialData](#initialData) using an explicit session and read its cluster time after the
+  last operation.
```

Comment:
@jyemin: I relocated the bulk of the logic to this section to keep "Executing a Test" lighter. This is modeled after "StaleDbVersion Errors on Sharded Clusters" below.